From cad0178cc0bdacd33f7faf2f86afa6d2a118f730 Mon Sep 17 00:00:00 2001
From: Kent Fredric <kentfredric@gmail.com>
Date: Wed, 14 Jun 2017 20:39:17 +1200
Subject: [PATCH] Fix for '.' removal from @INC in Perl 5.26

  require "foo.ext"

Never really implied it would load files from ./ , it only did so
as a side effect of '.' being in @INC.

The only way which works after 5.26 is to use an explicit ./  prefix,
which is special-cased for this purpose.
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index f6e2d49..c7ffd45 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -10,7 +10,7 @@ sub configure {
 
     # returns a reference to anonymous hash which is then interpreted as
     # additional options to the WriteMakeFile
-    $options = require "pam.cfg";
+    $options = require "./pam.cfg";
 
     if ( $Config{'osname'} eq 'solaris' && $Config{'osvers'} eq '2.6') {
       print "Adding a workaround for a bug in the Solaris 2.6 pam library\n";
-- 
2.13.1

